addons/mass_mailing/data/mass_mailing_demo.xml

```diff
@@ -81,7 +81,7 @@
             <tr>
                 <td class="o_mail_no_resize o_not_editable" style="text-align:left;"> </td>
```

Comment:
Commit message:

Added some tests to ensure that t-snippet and t-snippet-call really have
**there template** key as data-snippet

Adapted the views to the data-snippet changes adding
**data-snipet="tmpl_key".**